ci/install.sh

```diff
@@ -26,7 +26,7 @@
 # ------------------------------------------------------------------------------
 
 # Catch if last commit come from last travis build to prevent loop
-if [[ $TRAVIS_COMMIT_MESSAGE != *"**beta**"* && $TRAVIS_COMMIT_MESSAGE != *"**version**"* && $TRAVIS_COMMIT_MESSAGE != *"**CHANGELOG.md**"* ]]; then
+if [[ $TRAVIS_COMMIT_MESSAGE != *"chore(release):"* && $TRAVIS_COMMIT_MESSAGE != *"**beta**"* && $TRAVIS_COMMIT_MESSAGE != *"**version**"* && $TRAVIS_COMMIT_MESSAGE != *"**CHANGELOG.md**"* ]]; then
```

Comment:
Reeplace `chore` to `build`

Reply:
This is the default commit generated with "conventional-github-releaser" I'm going to check if can be custom or make a --amend to git